graphyte.py

```diff
@@ -64,31 +64,41 @@ def stop(self):
             self._thread.join()
             self.interval = None
 
-    def build_message(self, metric, value, timestamp):
+    def build_message(self, metric, value, timestamp, tags):
```

Comment:
This is a "public" function (no underscore prefix), so I'd prefer it if the signature was backwards-compatible (that's the reason it broke the tests). For example, if folks are subclassing and relying on the existing signature, it'll break for them. Can we make this have an empty default as well, like `tags=[]`?

Reply:
Done, but with a default dict type.